src/main/java/frc/team2412/robot/commands/launcher/LauncherPreSetAngle.java

```diff
@@ -0,0 +1,27 @@
+package frc.team2412.robot.commands.launcher;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+
+public class LauncherPreSetAngle extends Command {
```

Comment:
This is named like a thing/noun instead of a action verb, which is confusing. Consider naming this `SetLauncherAngle` instead.

Reply:
Updated, ty!

Reply:
? The class name hasn't changed...